packages/react-conformance-make-styles/src/matchers/toHaveMergeClassesCalledTimesWithClassName.test.ts

```diff
@@ -1,4 +1,4 @@
-import { mergeClasses } from '@fluentui/make-styles';
+import { mergeClasses } from '@fluentui/react-make-styles';
```

Comment:
These tests should use imports from `react-*` package as this package have only in in dependencies

Reply:
don't understand, why can't these tests depend on Griffel ?

Reply:
oh I see you're wrapping Griffel with `react-make-styles`?

Reply:
> oh I see you're wrapping Griffel with react-make-styles?

Yes, as we have many packages I will replace them one by one.

> don't understand, why can't these tests depend on Griffel ?

It's mistake that there was `@fluentui/make-styles` as is not in dependencies:

https://github.com/microsoft/fluentui/blob/8dfa712156b70414205b87b5b6d099367b0c297d/packages/react-conformance-make-styles/package.json#L30

They should depend on `@griffel/react`, but it's not a thing yet.